crates/bevy_render/src/render_graph/base.rs

```diff
@@ -128,20 +151,35 @@ pub(crate) fn add_base_graph(config: &BaseRenderGraphConfig, world: &mut World)
                                                           * bit depth for better performance */
                     usage: TextureUsage::OUTPUT_ATTACHMENT,
                 },
+                None,
+                None,
             ),
         );
     }
 
     if config.add_main_pass {
-        let mut main_pass_node = PassNode::<&MainPass>::new(PassDescriptor {
-            color_attachments: vec![msaa.color_attachment_descriptor(
+        let color_attachments = if config.add_post_pass {
+            vec![RenderPassColorAttachmentDescriptor {
+                attachment: TextureAttachment::Input("color_attachment".to_string()),
+                resolve_target: None,
+                ops: Operations {
+                    load: LoadOp::Clear(Color::rgb(0.1, 0.2, 0.3)),
```

Comment:
Isn't there a resource for clear color we could use here?

Reply:
Yes. Good point.